### ISSUE TITLE
Error get info about block

### ISSUE THREAD

**Issue by oggy95** [tronprotocol/java-tron#4348]
### 1. What did you do?

curl --request POST \
  --url http://3.225.171.164:8090/wallet/getblockbynum \
  --header 'Content-Type: application/json' \
  --data '{
	"num": 40134306
}'

Only this block on all available tron full nodes.
### 2. What did you expect to see?
Info about block

### 3. What did you see instead?
{
	"Error": "class com.alibaba.fastjson.JSONException : illegal state. 0"
}

**Comment by alecalve** [comment]
I can confirm I am running into the same issue:

```
$ curl -X POST -H "Content-Type: application/json" -d '{"num":40134306}' http://localhost:8091/walletsolidity/getblockbynum
{"Error":"class com.alibaba.fastjson.JSONException : illegal state. 0"}
```

**Comment by Denis-score** [comment]
Same issue on our side with 4.4.4 node version

**Comment by alecalve** [comment]
Our affected node is 4.4.3

**Comment by profforg** [comment]
Same issue on 4.4.4 version

**Comment by prih** [comment]
same, v4.4.4

**Comment by prih** [comment]
4.4.2, 4.4.3 - same

**Comment by lutk** [comment]
I confirm, the same result on the same block.

**Comment by athamidn** [comment]
same issue :(

**Comment by slashramin** [comment]
confirm that
the same result on the same block.
json rpc by blocknumber by block hash does't work

**Comment by prih** [comment]
the next blocks after 40134306 - is ok

**Comment by androzd** [comment]
Same block, same problem. Tried by number and hash.
Version 4.4.4
Multiple nodes

**Comment by prih** [comment]
it would be nice to be able to roll back transactions to a specific block... and resync

**Comment by kakelyo** [comment]
The error root cause is below txid decode failed:
456c74cba32be191fc385fb0dda55c5ec065ea423d172d102c782e709279d57d

**Comment by prih** [comment]
v4.4.5 - fixed!

**Comment by vorotech** [comment]
Confirmed, v4.4.5 did the trick

**Comment by juliajang** [comment]
Does anyone know what specifically was done in `v4.4.5` to fix this issue?

**Comment by joker-gdb** [comment]
It seems that they modified the request API to remove the invalid fields

**Comment by ethan1844** [comment]
Close this issue as it is already solved in [GreatVoyage-v4.4.5](https://github.com/tronprotocol/java-tron/releases/tag/GreatVoyage-v4.4.5).
